rllib/agents/a3c/a3c_torch_policy.py

```diff
@@ -90,9 +71,10 @@ def _value(self, obs):
     get_default_config=lambda: ray.rllib.agents.a3c.a3c.DEFAULT_CONFIG,
     loss_fn=actor_critic_loss,
     stats_fn=loss_and_entropy_stats,
-    postprocess_fn=add_advantages,
+    postprocess_fn=postprocess_ppo_gae,
```

Comment:
Probably replace ppo_gae with gae since it is now for multiple agents

Reply:
+1